### ISSUE TITLE
Two eero networks with the same SSID cannot be individually selected

### ISSUE THREAD

**Issue by rsnodgrass** [schmittx/home-assistant-eero#1]
If a user has TWO different eero networks, both of which share the same SSID, it is impossible to select one or the other in config_flow async_step_networks since the selection is based on the network name.  Since there are duplicate network names, selecting one then selects both.

**Comment by schmittx** [comment]
Good point, I don't have multiple networks personally so I haven't been able to test that directly.

Do you have multiple networks? If so, I'm curious how the Eero app displays multiple networks with the same SSID?

It's easy enough to add some additional identifier to the config flow list (similar to the config flow list for clients), but I'm not sure what's the best (i.e. city, region, postal code, ISP, etc.)?

**Comment by rsnodgrass** [comment]
Eero displays them as the same SSID too...which is annoying to know which one to select.  Though at least in that case they are positional in the app, so I know the first one is one location and the second another.  

The MAC address of the eero that is the main one (with the wired Internet connection) would be a suitable identifier, since IP could change, and address info could actually be similar.

**Comment by schmittx** [comment]
Gotcha, understood.

The MAC address for the gateway of each network should be straightforward, I'll start looking into this.

Another thought, it might be a little more user friendly to append the names of the Eero within each network (ex. `My Network SSID [Basement, Living Room, Office]`), assuming they're unique. Thoughts?

**Comment by schmittx** [comment]
A separate but related concern with 2 identically named SSIDs are the custom services for restart, IPV6, local DNS caching, etc. They all take `network_name` as an optional attribute.

I'm not sure what the best (simplest) identifiers for those would be if you wanted to target a single network. Any thoughts?

Side question, I'm curious what your use case is for having 2 networks with same SSID? Are they different physical locations?

**Comment by rsnodgrass** [comment]
Yes, they are at two different physical locations. It was just to save time setting up a dozen or so devices don't have to be resetup when moving between them...but I probably would have reduced some hair pulling selecting
  
​
  
​However, I probably should have just setup a separate SSID since eero's interface also only shows network selection by SSID. Thus I'm always making a semi-random guess selecting a physical location to manage. It is kind of annoying.

**Comment by schmittx** [comment]
Understood, your idea of not having to re-configure devices makes sense to me for sure.

I'll see what I can come up with to support this and be as user-friendly as possible, I've got some ideas but need to get into the details a bit.

To confirm, does each network show up as a different device (but with same name obviously)?

**Comment by rsnodgrass** [comment]
Yes, each network does show up as a different device.

**Comment by schmittx** [comment]
I just pushed a new release, give it a try and share any feedback.

In config and options flow, I added city/state as identifiers to the SSID if duplicate names are detected. I thought this was friendlier than gateway name/MAC. But this could be a problem if your physical locations are very close to each other, so let me know.

I also updated the custom services to accept network/eero names or IDs. The network ID can be found as an attribute to `sensor.<SSID>_ssid` entity.

Any suggestions/improvements are welcomed.
